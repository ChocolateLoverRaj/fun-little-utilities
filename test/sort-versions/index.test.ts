import sort from 'sort-versions'

it('matches snapshot', () => {
  const input = [
    '3.1.0',
    'v3.0.2',
    'invalid 0',
    '2.0.1',
    'v1.0.0',
    'invalid 1',
    '1.0.1',
    '0.1.2',
    'v0.0.0',
  ]

  expect(sort(input)).toMatchSnapshot()
})
