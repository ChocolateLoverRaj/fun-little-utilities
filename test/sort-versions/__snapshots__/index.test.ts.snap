// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
Array [
  "v0.0.0",
  "0.1.2",
  "v1.0.0",
  "1.0.1",
  "2.0.1",
  "v3.0.2",
  "3.1.0",
]
`;
